style/css/home.css

```diff
@@ -401,13 +401,14 @@ header {
 
 #about {
   height: auto;
-  max-height: 430px;
+  max-height: 369px;
   overflow: hidden; }
   #about .about-left {
     width: 50%;
     background-image: url("../../images/collaboration2.jpg");
     -webkit-background-image: url("../../images/collaboration2.jpg");
     -moz-background-image: url("../../images/collaboration2.jpg");
+    color: blue;
```

Comment:
Not at my normall cmputer so cant try this but im pretty sure this is a content bug.

Because theres no content in the div its not showing. So the right fix would be 

`content: ''`

To put some empty content in

Reply:
In HTML or CSS?

Reply:
Html